crates/rpc/rpc/Cargo.toml

```diff
@@ -75,7 +75,7 @@ tracing.workspace = true
 tracing-futures = "0.2"
 schnellru.workspace = true
 futures.workspace = true
-derive_more.workspace = true
+derive_more = { workspace = true, default-features = false, features = ["deref", "deref_mut", "constructor"] }
```

Comment:
could we do default-features = false on the workspace level?

because I believe this does not take effect otherwise

Reply:
indeed, will open issue